packages/circuit-ui/components/Popover/Popover.tsx

```diff
@@ -138,40 +136,74 @@ export const PopoverItem = ({
   );
 };
 
-const wrapperStyles = ({ theme }: StyleProps) => css`
-  label: popover;
+const wrapperBaseStyles = ({ theme }: StyleProps) => css`
   padding: ${theme.spacings.byte} 0px;
   border: 1px solid ${theme.colors.n200};
   box-sizing: border-box;
   border-radius: ${theme.borderRadius.byte};
   background-color: ${theme.colors.white};
+  z-index: ${theme.zIndex.popover};
+  visibility: hidden;
 
   ${theme.mq.untilKilo} {
+    transform: translateY(100%);
+    transition: transform ${theme.transitions.default},
+      visibility ${theme.transitions.default};
     border-bottom-right-radius: 0;
     border-bottom-left-radius: 0;
   }
 `;
 
-const PopoverWrapper = styled('div')(wrapperStyles, shadow);
+type OpenProps = { isOpen: boolean };
+
+const wrapperOpenStyles = ({ theme, isOpen }: StyleProps & OpenProps) =>
+  isOpen &&
+  css`
+    visibility: visible;
+
+    ${theme.mq.untilKilo} {
+      transform: translateY(0);
+    }
+  `;
+
+const PopoverWrapper = styled('div')<OpenProps>(
+  shadow,
+  wrapperBaseStyles,
+  wrapperOpenStyles,
+);
 
 const dividerStyles = (theme: Theme) => css`
   margin: ${theme.spacings.byte} ${theme.spacings.mega};
   width: calc(100% - ${theme.spacings.mega}*2);
 `;
 
-const Overlay = styled.div(
-  ({ theme }: StyleProps) => css`
+const overlayStyles = ({ theme }: StyleProps) => css`
+  ${theme.mq.untilKilo} {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: ${theme.colors.overlay};
+    z-index: ${theme.zIndex.popover}-1;
```

Comment:
The `- 1` needs to be inside the interpolation so that it can be evaluated in JavaScript (take a look at the snapshot, currently the value is `30-1` which is invalid).

```suggestion
    z-index: ${theme.zIndex.popover - 1};
```

Alternatively, you can do math calculations in CSS using the [`calc()`](https://developer.mozilla.org/en-US/docs/Web/CSS/calc()) method, though that would be a bit more verbose here.